src/GraphClient.ts

```diff
@@ -31,9 +32,10 @@ export default class GraphClient {
    * Documentation: https://www.apollographql.com/docs/react/advanced/subscriptions/
    *
    * @param {string} subscriptionQry Subscription query.
+   * @param {TransactionHandler} handler Transaction handler object.
    * @return {Subscription} Query subscription object.
    */
-  public subscribe(subscriptionQry: string): Subscription {
+  public subscribe(subscriptionQry: string, handler: TransactionHandler): Subscription {
```

Comment:
TransactionHandler is never going to change. It's always a singleton object. Shouldn't this be part of constructor.
While creating instance of GraphClient in facilitator start we can pass the handler object in the constructor. 
What do you think?

Reply:
This way also, it's a singleton. I think handler should be passed along with the query.